chapters/synchronization.txt

```diff
@@ -2651,8 +2645,7 @@ The handle types supported by pname:handleType are:
 ****
   * [[VUID-VkImportSemaphoreWin32HandleInfoKHR-handleType-01140]]
     pname:handleType must: be a value included in the
-    <<synchronization-semaphore-handletypes-win32,Handle Types Supported by
-    VkImportSemaphoreWin32HandleInfoKHR>> table.
+    <<synchronization-semaphore-handletypes-win32>> table.
```

Comment:
Please don't do this simplification (dropping the explicit xref text). While fine for the spec outputs themselves, it creates problems with extracted ref pages where they contain such xrefs.

Reply:
Alight, this is leftover anyway.
What I wanted here is the `VkImportSemaphoreWin32HandleInfoKHR` to be `tt`, but I failed to coerce the PDF style file to render it with correct (inherited from link color), and since then remove that.

Might be worth a styleguide note (if anybody reads that).
Anchors, and link labels cannot contain `*link:` as that results in a `<a>` tag inside a  `<a>` tag.
`*name` results in the above PDF problem.

Reply:
reverted as asked.